addons/web/static/src/legacy/js/libs/jquery.js

```diff
@@ -127,7 +127,7 @@ $.fn.extend({
      */
     closestScrollable() {
         let $el = this;
-        while ($el[0] !== document.scrollingElement) {
+        while ($el[0] !== $el[0].ownerDocument.scrollingElement) {
```

Comment:
I'll be doing that (but differently) with this PR: https://github.com/odoo/odoo/pull/98429 which contains a forward-port of this one https://github.com/odoo/odoo/pull/98514, let's avoid conflicts (especially since my version does not change this line and keeps its history): remove this commit, sorry :wink: